_pages/2019/x/psets/0/scratch.md

```diff
@@ -6,21 +6,16 @@ layout: 2019/x
 
 ## Heads-Up
 
-In lecture, David demonstrated Scratch 3, a new version of Scratch that MIT hasn't quite finished, otherwise known as a "beta." It lives at [beta.scratch.mit.edu](https://beta.scratch.mit.edu/). Even though it's not quite finished, it's better than Scratch 2, its predecessor, insofar as it's implemented in HTML, CSS, and JavaScript (languages that you'll soon know!) rather than Flash, an older technology that's disabled by default in most browsers these days.
+In lecture, David demonstrated Scratch 3, a new version of Scratch that MIT hadn't quite finished, otherwise known as a "beta." It's better than Scratch 2, its predecessor, insofar as it's implemented in HTML, CSS, and JavaScript (languages that you'll soon know!) rather than Flash, an older technology that's disabled by default in most browsers these days.
```

Comment:
I think we can reword, since I never showed URL or mentioned beta in lecture? Hopefully?

Reply:
I removed all mention of beta.